xls/examples/fill.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
is that design specific or can it be reused across many design? also why is it not referenced from the bazel rules?

Reply:
if it's PDK specific I think we can maintain it in the PDK integration in bazel_rules_hdl, similar to how we do it for tcl files for synthesis and sta.

Reply:
Yes, I think we can do that. I will work on adding this in bazel_rules_hdl

Reply:
For the record, here is the link to relevant PR in bazel_rules_hdl: https://github.com/hdl/bazel_rules_hdl/pull/173